### ISSUE TITLE
[9.0.2] build fails while unknown symbol

### ISSUE THREAD

**Issue by tuxmaster** [greenbone/gvm-libs#40]
/builddir/build/BUILD/gvm-libs-9.0.2/nasl/nasl_crypto2.c: In function 'nasl_aes128_gcm_encrypt':
/builddir/build/BUILD/gvm-libs-9.0.2/nasl/nasl_crypto2.c:1572:51: error: 'GCRY_CIPHER_MODE_GCM' undeclared (first use in this function)
   return encrypt_data (lexic, GCRY_CIPHER_AES128, GCRY_CIPHER_MODE_GCM);
                                                   ^
/builddir/build/BUILD/gvm-libs-9.0.2/nasl/nasl_crypto2.c:1572:51: note: each undeclared identifier is reported only once for each function it appears in
/builddir/build/BUILD/gvm-libs-9.0.2/nasl/nasl_crypto2.c: In function 'nasl_aes256_gcm_encrypt':
/builddir/build/BUILD/gvm-libs-9.0.2/nasl/nasl_crypto2.c:1578:51: error: 'GCRY_CIPHER_MODE_GCM' undeclared (first use in this function)
   return encrypt_data (lexic, GCRY_CIPHER_AES256, GCRY_CIPHER_MODE_GCM);
                                                   ^
/builddir/build/BUILD/gvm-libs-9.0.2/nasl/nasl_crypto2.c: In function 'nasl_aes128_gcm_encrypt':
/builddir/build/BUILD/gvm-libs-9.0.2/nasl/nasl_crypto2.c:1573:1: error: control reaches end of non-void function [-Werror=return-type]
 }
 ^
/builddir/build/BUILD/gvm-libs-9.0.2/nasl/nasl_crypto2.c: In function 'nasl_aes256_gcm_encrypt':
/builddir/build/BUILD/gvm-libs-9.0.2/nasl/nasl_crypto2.c:1579:1: error: control reaches end of non-void function [-Werror=return-type]
 }
 ^
cc1: all warnings being treated as errors
make[2]: *** [nasl/CMakeFiles/openvas_nasl_shared.dir/nasl_crypto2.c.o] Error 1

For the full build log, see the attachments. 

[build.log](https://github.com/greenbone/gvm-libs/files/1796698/build.log)
[hw_info.log](https://github.com/greenbone/gvm-libs/files/1796699/hw_info.log)
[installed_pkgs.log](https://github.com/greenbone/gvm-libs/files/1796700/installed_pkgs.log)
[root.log](https://github.com/greenbone/gvm-libs/files/1796701/root.log)
[state.log](https://github.com/greenbone/gvm-libs/files/1796702/state.log)

**Comment by tuxmaster** [comment]
It looks like, cmake don't check for the right version of libgcrypt.
gvm-lib only check for presence not for version.
The needed macro was added at 1.6

**Comment by wiegandm** [comment]
The feature branch adding support for a version check and setting the minimum required version to 1.6 was merged into the `openvas-libraries-9.0` branch in 37d72c2d4358cc21e2e3ad2d05b587a06e67e8be, closing here.
